R/run.R

```diff
@@ -663,7 +691,11 @@ CmdStanMCMCProcs <- R6::R6Class(
           # (Note: state 2 is only used because rejection in cmdstan is printed
           # to stdout not stderr and we want to avoid printing the intial chain metadata)
           next_state <- state
-          if (state < 3 && regexpr("Rejecting initial value:", line, perl = TRUE) > 0) {
+          if (state < 3 && regexpr("refresh =", line, perl = TRUE) > 0) {
```

Comment:
Once we find `refresh = 0`, we wait for a blank line and that marks the end of "metadata". This fixes the "sig_figs = -1" prints bug.